tests/integration/test_dss.py

```diff
@@ -108,6 +141,41 @@ def test_create_notebook(cleanup_after_initialize) -> None:
     assert deployment.status.availableReplicas == deployment.spec.replicas
 
 
+@pytest.mark.gpu_run
+def test_notebook_gpu_availability(notebook_image, cleanup_after_initialize):
```

Comment:
does `notebook_image` get used anywhere?  I think it might be left over from a previous iteration?